pkg/await/await.go

```diff
@@ -42,22 +42,36 @@ import (
 // --------------------------------------------------------------------------
 
 const (
-	appsV1Deployment                     = "apps/v1/Deployment"
-	appsV1Beta1Deployment                = "apps/v1beta1/Deployment"
-	appsV1Beta2Deployment                = "apps/v1beta2/Deployment"
-	autoscalingV1HorizontalPodAutoscaler = "autoscaling/v1/HorizontalPodAutoscaler"
-	extensionsV1Beta1Deployment          = "extensions/v1beta1/Deployment"
-	storageV1StorageClass                = "storage.k8s.io/v1/StorageClass"
-	coreV1ConfigMap                      = "v1/ConfigMap"
-	coreV1LimitRange                     = "v1/LimitRange"
-	coreV1Namespace                      = "v1/Namespace"
-	coreV1PersistentVolume               = "v1/PersistentVolume"
-	coreV1PersistentVolumeClaim          = "v1/PersistentVolumeClaim"
-	coreV1Pod                            = "v1/Pod"
-	coreV1ReplicationController          = "v1/ReplicationController"
-	coreV1ResourceQuota                  = "v1/ResourceQuota"
-	coreV1Secret                         = "v1/Secret"
-	coreV1Service                        = "v1/Service"
+	appsV1Deployment                            = "apps/v1/Deployment"
+	appsV1Beta1Deployment                       = "apps/v1beta1/Deployment"
+	appsV1Beta2Deployment                       = "apps/v1beta2/Deployment"
+	autoscalingV1HorizontalPodAutoscaler        = "autoscaling/v1/HorizontalPodAutoscaler"
+	extensionsV1Beta1Deployment                 = "extensions/v1beta1/Deployment"
+	storageV1StorageClass                       = "storage.k8s.io/v1/StorageClass"
+	coreV1ConfigMap                             = "v1/ConfigMap"
```

Comment:
I _thought_ these were projected as `core/v1/...`.  Does this PR change this?

Reply:
This PR does _not_ change this. Pulumi refers to these as `core/v1/...`, but the API server refers to them as `v1/...`. The await logic was built on top of the raw types exposed by the API server, rather than our projection of those types, which is why we refer to the API types using these non-fully-qualified names.